### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Jasonfintech**
a link

### CODE REVIEW

**Comment by Jasonfintech** [comment]
@laosb

**Comment by laosb** [comment]
I'll review it now.

File: index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
   </head>
   <body>
     <h1>YGDO</h1>
+    <a href="http://www.w3school.com.cn">This is a link</a>
```
Comment (laosb) [inline index.html]: What's that website?

Comment (laosb) [inline index.html]: Please change it to `nbyg.net`. And then LGTM.

**Comment by Jasonfintech** [comment]
@laosb ready to review.sorry

**Comment by laosb** [comment]
I think it's good to merge.

**Pull request merged**